brage-import/src/main/java/no/sikt/nva/brage/migration/lambda/cleanup/MainMethodClassForDeletionOfImportedPublications.java

```diff
@@ -0,0 +1,48 @@
+package no.sikt.nva.brage.migration.lambda.cleanup;
+
+import static no.unit.nva.publication.s3imports.ApplicationConstants.defaultS3Client;
+import static org.mockito.Mockito.mock;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.io.Writer;
+import no.unit.nva.publication.service.impl.ResourceService;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.ioutils.IoUtils;
+
+@JacocoGenerated
+public class MainMethodClassForDeletionOfImportedPublications {
+
+    /**
+     * Required environment variables are: TABLE_NAME, EVENT_BUS_NAME, EVENTS_BUCKET, API_HOST, AWS_REGION. Put s3
+     * location uri instead of empty string and run main method
+     */
+
+    public static final String S3LOCATION = "{ 'uri' : 's3://brage-migration-reports-884807050265/HANDLE_REPORTS/2022-12-13:13/'}";
+    public static final Context context = mock(Context.class);
+
+    public static void main(String[] args) {
+        var s3Client = defaultS3Client();
+        var getHandler = new ListImportedBragePublicationsHandler(s3Client);
+        var deleteHandler = new DeleteImportedBragePublicationHandler(ResourceService.defaultService());
+        var identifiers = getHandler.handleRequest(IoUtils.stringToStream(S3LOCATION), context);
+        identifiers.forEach(identifier -> {
+            try {
+                deleteHandler.handleRequest(IoUtils.stringToStream(toJsonString(identifier)), context);
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+        });
+    }
```

Comment:
Think you should take a look at https://docs.aws.amazon.com/cli/latest/reference/lambda/invoke.html on how to invoke a lambda from aws-cli. Right now you are running the lambda and resourceService locally on your computer.

Reply:
Her er også en ressurs om hvordan man sparker igang lambdas from java code:https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/examples-lambda.html

Reply:
Lambda should be invoked by a single identifier, right? I can use cli, but only by invoking it by single request with single identifier - this is not an option, right? For now it will not deserialize the list of identifiers. I can invoke it with cli, but as said before - only with single identifier. That is the purpose of this Main class - to delete all imported publications by one click, alternatively a can create third function that does the same, than it will be possible to invoke it by cli or aws console